dependencies.props

```diff
@@ -84,6 +84,14 @@
       <Version>1.0.0-prerelease-00704-03</Version>
     </StaticDependency>
 
+    <!--
+      The NETNative targeting pack is in the projectn-tfs build-info, but auto-upgrade isn't wanted.
+      Verify it at a static version rather than referencing the build-info.
+    -->
+    <StaticDependency Include="Microsoft.TargetingPack.Private.NETNative">
+      <Version>1.0.1-beta-24430-00</Version>
```

Comment:
This works in master but in the dev/api branch we currently have 2 different versions of this package until @SedarG gets the TP work done to combine them.